Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonThemeComboBox.cs

```diff
@@ -1,9 +1,12 @@
 ﻿#region BSD License
 /*
- * 
+ *
```

Comment:
@Wagnerp When did you create this control. It's not part of the Original Toolkit ??

Reply:
That would have been last year. It can be removed if necessary.

Reply:
@Wagnerp 
That's OK, I'll change the license header on the go....